source/includes/_feature-analyzing.md

```diff
@@ -250,3 +250,76 @@ The result will be an array of output cubes:
     }
 }
 ```
+
+### Transforming analyses for presentation
+
+The `transform` member of an analysis specification (or multitable definition)
+provides a way to transform that dimension's results after computation. The 
+cube result dimension will always be derived from the `variable_query` part of the
+request (`{variable: $variableId})`, `{function: f, args: [$variableId, …]}`, &c.
+
+### Structure
+
+A `transform` is an array of target transforms for **valid** output-dimension elements. 
+Therefore to create a valid `transform` it is generally necessary to make a cube query, inspect
+the result dimension, and proceed from there. For categorical and multiple response
+variables, elements may also be obtained from the variable entity.
+
+Consider the following example result dimension:
+
+| Name       | missing |
+|------------|---------|
+| Element A  |         |
+| Element B  |         |
+| Element C  |         |
+| Don’t know |         |
+| Not asked  | true    |
+
+A full `transform` can specify a new order of output elements, new names, 
+and in the future, bases for hypothesis testing, result sorting, and 
+aggregation of results. A `transform` will have **four** elements, which may
+contain keys: 
+
+- **i**: index or array of indices in the target row/column
+- **name**: name of new target column
+- **sort**: `-1` or `1` indicating to sort results descending or ascending by this element
```

Comment:
I don't see how "sort" can work like this if we're specifying the order on the dimensions explicitly.